dev-libs/libclc/libclc-0.2.0_pre20180610.ebuild

```diff
@@ -63,10 +67,18 @@ pkg_setup() {
 }
 
 src_configure() {
+	local LIBCLC_TARGETS=()
+
+	use video_cards_nvidia && LIBCLC_TARGETS+=("nvptx-- nvptx64-- nvptx--nvidiacl nvptx64--nvidiacl")
+	use video_cards_r600 && LIBCLC_TARGETS+=("r600--")
+	use video_cards_radeonsi && LIBCLC_TARGETS+=("amdgcn-- amdgcn-mesa-mesa3d amdgcn--amdhsa")
+
+	[ ${#LIBCLC_TARGETS[@]} -ne 0 ] || die "libclc target missing!"
```

Comment:
`[[ ${libclc_targets[@]} ]] || ...` is the canonical way. That said, this can't really happen, can it?

Reply:
Not with the REQUIRED_USE. I used it as an assert to make sure the array manipulation worked as expected.